src/functions/module_lookup.ts

```diff
@@ -547,15 +558,22 @@ export const XLOOKUP = {
 
     const rangeLen = lookupDirection === "col" ? lookupRange[0].length : lookupRange.length;
 
-    const mode = _matchMode === 0 ? "strict" : _matchMode === 1 ? "nextGreater" : "nextSmaller";
+    const mode =
+      _matchMode === 0
+        ? "strict"
+        : _matchMode === 1
+        ? "nextGreater"
+        : _matchMode === -1
+        ? "nextSmaller"
+        : "wildcard";
```

Comment:
can we write this differently with a simple object mapping ? It would be easier to read I think
```ts
const MATCH_MODE = {
  0: "strict",
  1: "nextGreater",
  ...
}
```